readthedocs/api/base.py

```diff
@@ -39,7 +39,7 @@ class Meta(object):
         allowed_methods = ['get', 'post', 'put']
         queryset = Project.objects.api()
         authentication = PostAuthentication()
-        authorization = DjangoAuthorization()
+        authorization = ReadOnlyAuthorization()
```

Comment:
Authorization is already default to `ReadOnlyAuthorization`, I'm just being explicit here.